### DISCUSSION TITLE
Guida in Spagnolo

### DISCUSSION THREAD

**Discussion by gbvitrano** [ondata/guidaPraticaPubblicazioneCSV#10] [Idee/Proposte]
In spagnolo la guida è solo in pdf? 
Perché non facciamo una guida multi lingua... ? per lo spagnolo dobbiamo fare solo copia incolla :-) 
ODS alla conquista della Spagna e America Latina

**Comment by aborruso** [comment]
È un ottima idea. Predisponi la cosa?

**Comment by gbvitrano** [comment]
ok, mi inizio a copiare il testo :-)
mi correggo onData è ODS alla conquista della Spagna e America Latina :-) :-)

**Comment by gbvitrano** [comment]
Hola la version en español esta completa, la mayor parte del trabajo está hecho. La Asociación onData habla español 😀😀😀 https://gbvitrano.github.io/guidaPraticaPubblicazioneCSV-ES/
